utilities/xmlrpcpp/test/test_socket.cpp

```diff
@@ -0,0 +1,1368 @@
+
```

Comment:
Copyright block.

Reply:
Which license should I use here? I tend towards BSD but the rest of the library is licensed LGPL.

Reply:
Either one would be fine. Embedding BSD inside an LGPL project is not a problem, though you should update the package.xml to declare both. If everything else is LGPL it's simpler/"cleaner" to follow the exisiting projects license. So I guess I'd suggest defaulting to BSD if you're ok with it and it's more liberal.

Reply:
Ok; added an LGPL copyright block to all of the files that I've added in this and the previous PR.